homeassistant/components/tuya/select.py

```diff
@@ -274,6 +274,24 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Air Purifier
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf46h2s6dzm
+    "kj": (
+        SelectEntityDescription(
+            key=DPCode.COUNTDOWN,
+            name="Countdown",
+            device_class=TuyaDeviceClass.COUNTDOWN_MINUTES,
```

Comment:
This is not guaranteed to be minutes

Reply:
We should use the one you've made for hours, and not translate if it is just a number. 
I've seen occurrences it meant x10 (so, 1=10, 2=20, but also 1=1 hour)

```suggestion
            device_class=TuyaDeviceClass.COUNTDOWN,
```

Reply:
But we shouldn't use the DeviceClass of the countdown which is defined in the documentation in hours. Or should we?

How about we create one that lists the number in just `Timer 1`, `Timer 2`... just to improve the user experience?

Reply:
> But we shouldn't use the DeviceClass of the countdown which is defined in the documentation in hours. Or should we?

The documentation shows a default offered, however, manufacturers can fill it in their selfs and change it to their likings, hence it is not strictly defined.

Reply:
So there shouldn't be DeviceClass for countdown? Or should we create a general with `Timer 1, Timer 2, Timer 3...`?

Reply:
So there shouldn't be DeviceClass for countdown? Or should we create a general with `Timer 1, Timer 2, Timer 3...`?

Reply:
I think your countdown class for the documented default is nice and sane. This would still work if it has 1, 2, 3, a, v, b, d, e, f, g, or whatever :)

Reply:
Got it... you're right.